packages/jest-worker/src/WorkerPool.ts

```diff
@@ -16,15 +16,6 @@ import type {
   WorkerPoolInterface,
 } from './types';
 
-const canUseWorkerThreads = () => {
-  try {
-    require('worker_threads');
-    return true;
-  } catch {
-    return false;
-  }
-};
```

Comment:
Looks like this is Node 10 specific. `worker_threads` are stable since [node v12.11.0](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V12.md#2019-09-25-version-12110-current-bridgear)